src/components/gui/gui.jsx

```diff
@@ -73,6 +74,7 @@ const GUIComponent = props => {
             {feedbackFormVisible ? (
                 <FeedbackForm />
             ) : null}
+            {(window.WebGLRenderingContext) ? null : (<WebGlModal />)}
```

Comment:
This doesn't get called as WebGL errors cause the error-boundary to get used.

If the renderer starts handling the errors and failing by raising its own error, then this is unnecessary as it'll still get handled by the error-boundary. But if the renderer handles the errors, skips the stage, but doesn't otherwise fail we'll still want to show the unsupported modal.